### ISSUE TITLE
[PATCH] i18n and Japanese Translation

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#67]
```
Hi,

PhoneGap plugin is not i18ned. I wrote the patch to externalize the messages 
and added Japanese translation.

Could you apply the i18n and Japanese Translation?

best regards,

Takashi Okamoto


```

Original issue reported on code.google.com by `toran...@gmail.com` on 1 Feb 2012 at 5:00

Attachments:
- [i18n-and-japanese-translation.patch](https://storage.googleapis.com/google-code-attachments/mobile-web-development-with-phonegap/issue-67/comment-0/i18n-and-japanese-translation.patch)

**Comment by GoogleCodeExporter** [comment]
```
I've applied the patch to the svn trunk 
http://code.google.com/a/eclipselabs.org/p/mobile-web-development-with-phonegap/
source/detail?r=133. Assuming testing goes ok, it will be included in the 
upcoming 1.2.7 update.

Thanks for the contribution!  Arigatou gozaimasu
```

Original comment by `paul.beu...@gmail.com` on 1 Feb 2012 at 5:52
- Changed state: **Started**
- Added labels: **Milestone-Release1.2.7**, **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Validated in 1.27 that Japanese characters are written when 
eclipse/configuration/config.ini is updated with osgi.nl=ja
```

Original comment by `paul.beu...@gmail.com` on 7 Feb 2012 at 1:14

**Comment by GoogleCodeExporter** [comment]
```
Released with 1.2.7 February 7, 2012
```

Original comment by `paul.beu...@gmail.com` on 7 Feb 2012 at 4:10
- Changed state: **Fixed**
